pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -23,8 +23,13 @@ social:
   "twitter:player:height": "360"
 ---
 
+%link{href: "/css/highlights-homepage.css", rel: "stylesheet", id: "highlight-styles"}
+
 -facebook = {:u=>"http://#{request.host}/us"}
 
+-HIGHLIGHTS_CONSTANTS = [{:id=>16, :heading_s=>"President Obama is the first president to write a line of code, hosting a historic Hour of Code for students at the White House.", :short_heading_s=>"Coder in Chief", :image_url_s=>"obama-fistbump.jpg", :link_text_s=>"Watch the video", :url_s=>"https://youtu.be/AI_dayIQWV4"}, {:id=>19, :heading_s=>"From Shanghai to Sydney to Brussels to New York City.", :short_heading_s=>"Apple hosts Hour of Code workshops in every store worldwide", :image_url_s=>"apple-store-celebrate.jpg", :link_text_s=>nil, :url_s=>nil}, {:id=>45, :heading_s=>"The actress, activist, and entrepreneur hosted an Hour of Code with the entire staff at The Honest Company", :short_heading_s=>"Jessica Alba learns an Hour of Code", :image_url_s=>"jessica.jpg", :link_text_s=>"Share on Facebook", :url_s=>"https://www.facebook.com/Code.org/photos/pb.309754825787494.-2207520000.1453330310./795469473882691/"}]
```

Comment:
Though it is great that we are removing the DB calls, doing so is not without cost. Now, we need a DTP to change the page (rather than merely updating the DB). And  changes to the DB no longer update content, as might reasonably be expected.

Do we intend to keep this as-is long-term? If so, I have general concerns about hard-coding data in the DB into our code.